<launch>
    <arg name="config_path" default = "$(find feature_tracker)/../config/iPhone/logitech_config.yaml" />
	  <arg name="vins_path" default = "$(find feature_tracker)/../config/../" />
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
      <param name="video_device" value="/dev/video0" />
      <param name="image_width" value="640" />
      <param name="image_height" value="480" />
      <param name="pixel_format" value="yuyv" />
      <param name="camera_frame_id" value="usb_cam" />
      <param name="io_method" value="mmap"/>
      <remap from= "/usb_cam/image_raw" to="/cam0/image_raw" />
    </node>


    <node name="feature_tracker" pkg="feature_tracker" type="feature_tracker" output="log">
        <param name="config_file" type="string" value="$(arg config_path)" />
        <param name="vins_folder" type="string" value="$(arg vins_path)" />
    </node>
</launch>
